fastlane/swift/Runner.swift

```diff
@@ -83,6 +124,7 @@ extension Runner {
     func disconnectFromFastlaneProcess() {
         self.shouldLeaveDispatchGroupDuringDisconnect = true
         self.dispatchGroup.enter()
+        print("Enter \(#function) \(#line)")
```

Comment:
Either delete or 👇 ?

```suggestion
        verbose(message: "Enter \(#function) \(#line)")
```